### DISCUSSION TITLE
[VOTE] Release Apollo version 2.1.0

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4717] [General]
Hi All,

@apolloconfig/pmc @apolloconfig/committers This is a call for vote to release Apollo version 2.1.0.

Release Note:

 * https://github.com/apolloconfig/apollo/releases/tag/v2.1.0

Release Candidate:

 * https://github.com/apolloconfig/apollo/releases/download/v2.1.0/apollo-configservice-2.1.0-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.1.0/apollo-adminservice-2.1.0-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.1.0/apollo-portal-2.1.0-github.zip
 * sha1 checksums
   - 1dc833eca99d99819c634dcd36c42e20ae8b1a37  apollo-configservice-2.1.0-github.zip
   - ebcf2ec92fc5cfce1f32f334330be9853b06bc96  apollo-adminservice-2.1.0-github.zip
   - 6ff8bf6f5ef64ac6cb5d28f02a5303f3598f150c  apollo-portal-2.1.0-github.zip

Release Tag :

 * https://github.com/apolloconfig/apollo/releases/tag/v2.1.0

Release CommitID :

 * https://github.com/apolloconfig/apollo/commit/60bd514df37a8cb2268a5e945a862dc86bd81445

Guide to build the release from source:

 * https://www.apolloconfig.com/#/en/deployment/distributed-deployment-guide?id=_2212-building-from-source-code

Voting will start now (2023/02/01) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by klboke** [comment]
+1 Release this package

**Comment by lepdou** [comment]
+1

**Comment by kezhenxu94** [comment]
+1

**Comment by hezhangjian** [comment]
+1

**Comment by zouyx** [comment]
+1 accepted

**Comment by nobodyiam** [comment]
The vote to release Apollo version 2.1.0 has passed with five +1 and zero -1.
